sarpy/io/phase_history/cphd.py

```diff
@@ -10,10 +10,11 @@
 import numpy
 
 from sarpy.compliance import int_func, integer_types, string_types
-from .cphd1_elements.CPHD import CPHDType, CPHDHeader
+from .cphd1_elements.utils import binary_format_string_to_dtype
```

Comment:
I generally try not to use relative imports, but really doubt that it matters here one bit, since these will surely have the same relative positions.